WordPress/running_post_startup_scripts.md

```diff
@@ -15,27 +15,37 @@ Linux App Service architecture inherently has non-persistent storage i.e. file c
 ## How Startup script works?
 
 It is a bash script (in /home/dev/startup.sh) that is executed each time an app container starts and the changes made by startup commands remain constant even upon restart or scaling out to multiple app instances. The reason is that when an app container starts, it's file system in non-persistent storage has a default initial state defined by the underlying docker image. When startup script is executed, it may update files in non-persistent storage and upon restarting the app, these files revert back to the original state and startup script is executed which provides the same final state of files in non-persistent storage.
-A custom script has many use cases. The following are some scenarios for which you need a custom startup file  
+
+A custom startup script has many use cases. The following are some scenarios for which you need a startup script. Please note that your app service must be restarted for any changes in startup script to take affect.
 
 ## Update Nginx configuration
 
 Specifications for nginx are defined in /etc/nginx/conf.d/spec-settings.conf. You can update any of these settings using a startup script. The following command in startup script changes nginx configuration of max simultaneous connections to 15000.
```

Comment:
Configurations for nginx are defined in **/etc/nginx/conf.d/default.conf** and **/etc/nginx/conf.d/spec-settings.conf** files. You can update any of these files using a startup script. For instance, the following commands in startup script changes the nginx configuration for max simultaneous connections to 15000.